tests/unit/schema/wrappers/test_field.py

```diff
@@ -357,6 +357,9 @@ def test_field_name_kword_disambiguation():
     )
     assert frum_field.name == "frum"
 
+    mapping_field = make_field(name="mapping")
```

Comment:
I've noticed that in `gapic/utils/reserved_names.py` you've also added `ignore_unknown_fields` filed, should it be renamed to `ignore_unknown_fields_` in here?

Reply:
Yes, I should be rigorous about that :P This is me being lazy.